framework-profile.yaml

```diff
@@ -11,119 +11,96 @@ flavors:
     - common-packages
     - kairos-toolchain
     - systemd-base
-    - dracut-network-legacy
   debian-arm-rpi:
     - common-packages
     - kairos-toolchain
     - systemd-base
-    - dracut-network-legacy
   ubuntu:
     - common-packages
     - kairos-toolchain
     - systemd-base
-    - dracut-network-legacy
   ubuntu-arm-rpi:
     - common-packages
     - kairos-toolchain
     - systemd-base
-    - dracut-network-legacy
   ubuntu-20-lts-arm-nvidia-jetson-agx-orin:
     - common-packages
     - kairos-toolchain
     - systemd-base
-    - dracut-network-legacy-compat
   ubuntu-20-lts-arm-rpi:
     - common-packages
     - kairos-toolchain
     - systemd-base
-    - dracut-network-legacy-compat
   ubuntu-22-lts-arm-rpi:
     - common-packages
     - kairos-toolchain
     - systemd-base
-    - dracut-network-legacy-compat
   ubuntu-22-lts:
     - common-packages
     - kairos-toolchain
     - systemd-base
-    - dracut-network-legacy-compat
   ubuntu-20-lts:
     - common-packages
     - kairos-toolchain
     - systemd-base
-    - dracut-network-legacy-compat
   fips-systemd:
     - common-packages
     - kairos-toolchain-fips
     - systemd-base
-    - dracut-network-legacy-compat
   fedora:
     - common-packages
     - kairos-toolchain
     - systemd-base
-    - dracut-network-legacy-compat
   rockylinux:
     - common-packages
     - kairos-toolchain
     - systemd-base
-    - dracut-network-legacy-compat
   almalinux:
     - common-packages
     - kairos-toolchain
     - systemd-base
-    - dracut-network-legacy-compat
   opensuse-tumbleweed:
     - common-packages
     - kairos-toolchain
     - systemd-base
     - systemd-latest
-    - dracut-network-legacy
   opensuse-tumbleweed-arm-rpi:
     - common-packages
     - kairos-toolchain
     - systemd-base
-    - systemd-latest
-    - dracut-network-legacy
   opensuse-leap:
     - common-packages
     - kairos-toolchain
     - systemd-base
-    - systemd-latest
-    - dracut-network-legacy
   opensuse-leap-arm-rpi:
     - common-packages
     - kairos-toolchain
     - systemd-base
-    - systemd-latest
-    - dracut-network-legacy
-  alpine-arm-rpi:
+  alpine:
     - common-packages
     - kairos-toolchain
-    - opensuse-leap-kernel
     - openrc
-  alpine-opensuse-leap:
-    - common-packages
-    - kairos-toolchain
-    - openrc
-    - opensuse-leap-kernel
-  alpine-ubuntu:
+    - alpine-pure
+  alpine-arm-rpi:
     - common-packages
     - kairos-toolchain
-    - ubuntu-kernel
     - openrc
+    - alpine-pure-rpi
 # See https://github.com/kairos-io/packages/pull/67 for rationale
-dracut-network-legacy:
+alpine-pure:
```

Comment:
🤣 why not just alpine? without the `pure` don't want to start here a war between pure and impure distros at some point hahaha

Reply:
should be alpine-kernel xDD

alpine is already used up there for the distro name, this was a way to differ between the alpine-{ubuntu,opensuse} and the newer one, but that is now dropped xDDD

will change it to alpine-kernel or alpine-common or something more straightforward.